pom.xml

```diff
@@ -99,47 +114,9 @@
       </exclusions>
       <scope>test</scope>
     </dependency>
-
-    <!-- WIRED TEST RUNNER DEPENDENCIES -->
-    <dependency>
-      <groupId>com.atlassian.plugins</groupId>
-      <artifactId>atlassian-plugins-osgi-testrunner</artifactId>
-      <version>${plugin.testrunner.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>javax.ws.rs</groupId>
-      <artifactId>jsr311-api</artifactId>
-      <version>1.1.1</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>com.google.code.gson</groupId>
-      <artifactId>gson</artifactId>
-      <version>2.2.2-atlassian-1</version>
-    </dependency>
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
-      <version>1.7.12</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.seleniumhq.selenium</groupId>
-      <artifactId>selenium-firefox-driver</artifactId>
-      <version>2.53.1</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>commons-logging</groupId>
-      <artifactId>commons-logging</artifactId>
-      <version>1.2</version>
-      <scope>test</scope>
-    </dependency>
   </dependencies>
 
   <build>
-    <testSourceDirectory>${project.basedir}/src/test/java/ut</testSourceDirectory>
```

Comment:
Rather than separate the source like this, I've switch the setup to have Surefire (used to run unit tests) ignore everything under `src/test/java/it`, and have AMPS (which uses Failsafe to run integration tests) ignore everything that's _not_ under `src/test/java/it`.

(Feel free to not do it this way; this is just how we write our own plugins.)